### PR TITLE
created the left column text

### PR DESCRIPTION
**Pull request by PemaYT**
left text has been inputted

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,9 +7,27 @@
...
 </head>
+<style>
+    @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap');
+ </style>
 <body>
-    <h1>I am ready to start coding! :)</h1>
-    <button onclick="myFunction()">Try me</button>
+    <div class="container">
+        <div class="columnLeft">
+            <h1>Learn to code by<br>
```
Comment (Zyanvel) [inline index.html]: Hi! I would suggest to change the code like this:
```suggestion
            <h1>Learn to code by<br>watching others </h1>
```

File: styles.css [hunk 2]
```diff
@@ -1,20 +1,52 @@
...
-    color: goldenrod;
+    color: rgb(255, 255, 255);
 }
 
 body {
     background-image: url(./assets/bg-intro-desktop.png);
     background-color: hsl(0, 100%, 74%);
     width: 1440px;
-    height: 800px;
+    /* height: 800px; */
```
Comment (Zyanvel) [inline styles.css]: Nit: If you are not using this, could you please remove it?

File: styles.css [hunk 3]
```diff
@@ -1,20 +1,52 @@
...
 }
 
 @media screen and (max-width: 375px) {
     body {
         background-image: url(./assets/bg-intro-mobile.png);
         width: 375px;
-        height: 1000px;
+        /* height: 1000px; */
```
Comment (Zyanvel) [inline styles.css]: Nit: If you are not using this, could you please remove it?

File: styles.css [hunk 4]
```diff
@@ -1,20 +1,52 @@
...
     }
-}
\ No newline at end of file
+}
+
+.container {
+    /* display: flex; */
```
Comment (Zyanvel) [inline styles.css]: Nit: Could you please remove the extra code?

File: styles.css [hunk 5]
```diff
@@ -1,20 +1,52 @@
...
+    /* justify-content: center; */
+    padding: 20% 30% 20% 30%;
+    font-size: 16px;
+}
+
+.columnLeft {
+    /* padding: 5% 10% 5% 10%; */
```
Comment (Zyanvel) [inline styles.css]: Same here ☝️

**Review by Zyanvel — CHANGES_REQUESTED** [review]
Overall looks great! Just left some nit comments since this will help you to write cleaner code 😄 Let me know if you have any questions

**Review by Zyanvel — APPROVED** [review]
Looks good to me :)

**Review by soleyingunn — APPROVED** [review]
Looks good to me

**Pull request merged**